resume-buddy/src/main/java/com/google/sps/Match.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.util.List;
+
+/** Class that matches reviewers to reviewees */
+public class Match {
```

Comment:
Please add a private constructor to this class, so no one can create an instance of Match.

`private Match() {}`